routes/auth.js

```diff
@@ -50,14 +55,16 @@ router.post('/logout', async function (req, res, next) {
 
 /* Authenticated */
 // Return {status: true} or {status: false}
-// Calling this API will refresh the session
-router.post('/authenticated', async function (req, res, next) {
+router.post('/authenticated', async (req, res, _) => {
     try {
-        if (req.session.tokens) {
-            return res.status(200).send({status: true});
-        } else {
-            return res.status(200).send({status: false});
+        const auth = req.headers['authorization']
+        const token = auth && auth.split(' ')[1];
+        let status = false;
```

Comment:
I thought about this some more and this might be an even cleaner way of doing this to avoid the if else block:
`const status = Boolean((token && await verifyToken(token)) || (!token && req.session.tokens))`
I did some minor testing with this but please verify again if you are going to use this.

I'll leave it up to your preference if you want to change this or not, it seems to work fine as is

Reply:
Thanks. I will double check.